src/main/java/aeneas/views/BuildSelectLevelView.java

```diff
@@ -101,6 +108,57 @@ public void initialize(URL location, ResourceBundle resources) {
 
     JFXDepthManager.setDepth(fileList, 1);
 
+    refresh();
   }
 
+  public void refresh() {
+    fileList.getItems().clear();
+    
+    fileList.getItems().add(createNewLevelLabel);
+    Path defaultLevelPath = Paths.get(System.getenv("HOME"), ".aeneas-kabasuji");
+    try {
+      Stream<Path> files = Files.list(defaultLevelPath).filter(file -> file.getFileName().toString().endsWith(".kbs"));
+      files.forEach((file) -> {
+        ObjectInputStream ois = null;
+        try {
+          ois = new ObjectInputStream(new FileInputStream(file.toFile()));
+        } catch (IOException e) {
+          System.err.println("Error reading file '" + file + "'");
+        }
+
+        try {
+          Level l = (Level) ois.readObject();
+          levelMap.put(file.toString(), l);
+        } catch (NumberFormatException e) {
+          // Do something here to load custom levels
```

Comment:
Ok. That can come after this in a separate PR. Just look at my other comments and we'll be good